game-app/map-data/src/main/java/org/triplea/map/description/file/MapDescriptionYamlGenerator.java

```diff
@@ -79,32 +77,35 @@
     final Optional<File> writtenYmlFile =
         MapDescriptionYamlWriter.writeYmlPojoToFile(mapDescriptionYaml);
 
-    if (writtenYmlFile.isPresent() && propsFile.exists()) {
+    if (writtenYmlFile.isPresent() && Files.exists(propsFile)) {
       // clean up the properties file, it is no longer needed
-      propsFile.delete();
+      try {
+        Files.delete(propsFile);
+      } catch (final IOException exception) {
+        log.error("Failed to delete file " + propsFile.toAbsolutePath(), exception);
```

Comment:
I don't know if we want to get an error report about this. So perhaps this should be a warn.

Second, a user should also know that they need to delete the original props file (where it is, and that it needs to be deleted). There is a behavior as well to move items into a 'bad-map' folder, that could perhaps be even better than instructing the user to delete the props file.

Reply:
Good point, previously any errors would've been silent if I'm not missing anything here